Jenkinsfile_CNP

```diff
@@ -16,4 +16,4 @@ withPipeline(type, product, component) {
         currentBuild.description = "Success!"
         error 'Access management project should not be built on prod environment yet.'
     }
-}
\ No newline at end of file
+}
```

Comment:
revert - shouldn't be any changes at all in this file

Reply:
Dont know why this was added, try to remove blank line and it will not undo the blank space

Reply:
Just git revert.